src/Nethermind/Nethermind.Evm/IntrinsicGasCalculator.cs

```diff
@@ -92,7 +92,7 @@ private static long AccessListCost(Transaction transaction, IReleaseSpec release
                 else
                 {
                     throw new InvalidDataException(
-                        $"Transaction with an access list received within the context of {releaseSpec.Name}");
+                        $"Transaction with an access list received within the context of {releaseSpec.Name}. Eip-2930 is not enabled.");
```

Comment:
Hmmm, when a node is syncing from zero and is not yet switched and it validates a 2930 transaction whtn this should not throw, right?

Reply:
does it?

Reply:
Maybe, need to check consequences, created task.